### ISSUE TITLE
Add-on for HomeAssistant

### ISSUE THREAD

**Issue by skybarberom** [Despernal/Recordurbate-Docker#1]
As an Idea, it would be nice to see an integration for HomeAssistant to see the history bar where the room has been online. 
Probably it would be sufficient extract the script where the query to see if room is online.

This would give an idea of what should have been recorded and the history of time the room has been online from a graphical interface

**Comment by Despernal** [comment]
That would need to be taken up with the software author, I just help facilitate it into a docker container. You might try opening this same issue up on the authors repo

**Comment by skybarberom** [comment]
> That would need to be taken up with the software author, I just help facilitate it into a docker container. You might try opening this same issue up on the authors repo

yes, you are right. Anyway I managed to do it directly from HomeAssistant calling the Chaturbate APIs. If someone is interested I can share.
